proposals/collection-expression-arguments.md

```diff
@@ -31,13 +31,16 @@ This syntax was chosen as it:
 1. Keeps all information within the `[...]` syntax.  Ensuring that the code still clearly indicates a collection being created.
 2. Does not imply calling a `new` constructor (when that isn't how all collections are created).
 3. Does not imply creating/copying the values of the collection multiple times (like a postfix `with { ... }` might.
-4. Is both not subtle, while also not being excessively verbose.  For example, using `;` instead of `,` to indicate
+4. Does not contort order of operations, especially with C#'s consistent left-to-right expression evaluation ordering semantics.
+   For example, it does not evaluate the arguments used to construct a collection *after* evaluating the expressions used to
+   populate the collection.
+6. Is both not subtle, while also not being excessively verbose.  For example, using `;` instead of `,` to indicate
```

Comment:
Missing `5`.

Consider using `1.` for each bullet since the numbering will be generated when displayed.